Doc/whatsnew/3.8.rst

```diff
@@ -973,8 +1109,8 @@ The :func:`socket.if_nameindex()`, :func:`socket.if_nametoindex()`, and
 ssl
 ---
 
-Added :attr:`ssl.SSLContext.post_handshake_auth` to enable and
-:meth:`ssl.SSLSocket.verify_client_post_handshake` to initiate TLS 1.3
+Added :attr:`~ssl.SSLContext.post_handshake_auth` to enable and
```

Comment:
The module is known from the context, but it is not clear what classes the attribute and the method belong to. Maybe you meant ``` :attr:`SSLContext.post_handshake_auth <ssl.SSLContext.post_handshake_auth>` ```?

There are similar issues with some other uses of the tilde. It lefts only the last component.